students/15th/mina/styles/login.css

```
@@ -0,0 +1,117 @@
+@import url('https://fonts.googleapis.com/css2?family=Lobster&display=swap');
+
+
+
+
+body { 
+    background-color: #fafafa;
+}
+.loginpage_wrap {
+    width: 400px; 
+    margin: 0 auto;
+}
```

로그인 창이 화면 정 중앙에 정렬하는 방법은 여러 가지가 있지만
아래와 같이 flex를 이용해서 쉽게 구현하실 수 있습니다.
flex를 이용하면 margin: 0 auto는 안 쓰셔도 됩니다.

그리고 height에 100vh는 view height를 뜻합니다. 브라우저의 높이만큼 height 값을 줘서 body 전체를 늘려서 세로로 가운데 정렬도 될 만큼의 세로 높이의 공간을 확보해줍니다.

```suggestion
body { 
    height: 100vh;
    display: flex;
    justify-content: center;
    align-items: center;
    background-color: #fafafa;
}
.loginpage_wrap {
    width: 400px; 
}
```